third_party/blink/renderer/core/DEPS

```diff
@@ -1,4 +1,8 @@
 include_rules = [
```

Comment:
please create new DEPS files for each subdirectory and be as narrow as possible for each directory

Reply:
I moved brave_session_cache.{cc,h} to a separate subdirectory, so that the DEPS file is dedicated just to the needs of those files.